dataactbroker/scripts/update_detached_procurement.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from dataactcore.logging import configure_logging
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.models.jobModels import Submission # noqa
+from dataactcore.models.userModel import User # noqa
+from dataactvalidator.health_check import create_app
+
+logger = logging.getLogger(__name__)
+
+
+def update_table(sess, type):
+    logger.info('updating ' + type)
+    invalid = sess.execute("select * from detached_award_procurement where " + type + "_agency_code='999'")
+    invalid_count = len(invalid.fetchall())
```

Comment:
Do we really want to do a fetchall() here? I believe that actually grabs all the rows. Is there any need for that since we don't use them? It might make more sense to just run a count query, especially over selecting * from the FPDS table (which has WAY too many columns)

Reply:
changed it to `count(*)`